cmds/webboot/types.go

```diff
@@ -1,25 +1,34 @@
 package main
 
-import ui "github.com/gizak/termui/v3"
+import (
+	ui "github.com/gizak/termui/v3"
+	"github.com/u-root/webboot/pkg/menu"
+)
 
 const (
 	tcURL    = "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"
 	wbtcpURL = "https://github.com/u-root/webboot-distro/raw/master/iso/tinycore/10.x/x86_64/release/TinyCorePure64.iso"
 )
 
 // bookmark record the list of iso webboot allow user to download
-var bookmark = []*BookMarkISO{
-	&BookMarkISO{
-		url:   tcURL,
-		label: "Download Tinycore v10.1",
-		name:  "TinyCorePure64-10.1.iso",
-	},
-	&BookMarkISO{
-		url:   wbtcpURL,
-		label: "Webboot Tinycorepure",
-		name:  "TinyCorePure64.iso",
-	},
-}
+var (
+	bookmark = map[string]([]*BookMarkISO){
+		"bookmark Tinycore1": []*BookMarkISO{
```

Comment:
I think since the map is already called bookmark you don't need to have 'bookmark' in the keys here